apps/easypid/base.app.config.js

```diff
@@ -0,0 +1,187 @@
+const APP_VARIANT = process.env.APP_VARIANT || 'production'
+
+const variants = {
+  development: {
+    bundle: '.dev',
+    name: ' (Dev)',
+  },
+  preview: {
+    bundle: '.preview',
+    name: ' (Preview)',
+  },
+  production: {
+    bundle: '',
+    name: '',
+  },
+}
+
+const variant = variants[APP_VARIANT]
+
+if (!variant) {
+  throw new Error(`Invalid variant provided: ${process.env.APP_VARIANT}`)
+}
+
+// NOTE: Keep this in sync with the `QrTypes` enum
+const baseInvitationSchemes = [
+  'openid',
+  'openid-initiate-issuance',
+  'openid-credential-offer',
+  'openid-vc',
+  'openid4vp',
+  'eudi-openid4vp',
+  'mdoc-openid4vp',
+  'id.animo.ausweis',
+  'id.animo.paradym',
```

Comment:
Sorry i should have been more clear. The app id should only be added to the respective app, so ausweis to funke wallet, paradym to paradym wallet